endpoints-msp-sessions/src/main/scala/com/thenewmotion/ocpi/sessions/SessionsService.scala

```diff
@@ -0,0 +1,33 @@
+package com.thenewmotion.ocpi.sessions
+
+import com.thenewmotion.ocpi.msgs.GlobalPartyId
+import com.thenewmotion.ocpi.msgs.v2_1.Sessions.{Session, SessionId, SessionPatch}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/**
+  * All methods are to be implemented in an idempotent fashion.
+  */
+trait SessionsService {
+
+  /**
+    * @return true if the session has been created and false if it has been updated
```

Comment:
A dedicated type would be better, I think.

Reply:
Agreed, I based it it other modules.  I'll update it also in those then for consistency.